### PR TITLE
feat: added vault filter flyout

### LINKED ISSUE #1193: Refactor outline vault to accomdate both credential and vault flyout
TODO: Refactor outline vault to accomdate both credential and vault flyout

Credential icon - no tooltip and border and width is different 

Make this compose-able

### LINKED ISSUE #1258: Populate flyout menu with the created list of vaults

### LINKED ISSUE #1270: Populate All Vault in the credential dropdown with the list of the vaults and update the name as selected in the credential

### PR DESCRIPTION
**Pull request by birtony**
Fixes #1193
Fixes #1258
A part of #1270

Improved `Flyout` component, added vault filter flyout.

https://user-images.githubusercontent.com/33902374/145371773-b635c8e6-10c3-42dd-8686-be03b2caeeef.mov

Signed-off-by: Anton Biriukov <anton.biriukov@securekey.com>

### CODE REVIEW

**Review by rolsonquadras — APPROVED** [review]

File: cmd/wallet-web/src/components/Flyout/FlyoutMenu.vue [hunk 1]
```diff
@@ -10,23 +10,23 @@
       right-0
       z-20
       mt-2
-      w-max
+      w-48
```
Comment (talwinder50) [inline cmd/wallet-web/src/components/Flyout/FlyoutMenu.vue:13]: As per the design on the smaller screen this width is 10 rem, this is now always 12 rem, Can you check once how this is rendered on smaller screens?

Comment (talwinder50) [reply cmd/wallet-web/src/components/Flyout/FlyoutMenu.vue:13]: https://app.abstract.com/projects/9ec92568-60a2-4fee-8215-5aa997d52428/branches/dc10bcee-d9a1-44f9-b644-8f843c6f1a19/commits/latest/files/95d5757f-f10e-43f6-ad50-0faa236217fd/layers/39E482B1-E229-4C49-87A6-572E76E43F6F?collectionId=ba692773-75df-473a-a41c-38a0647809a4&collectionLayerId=0d8e1432-1eb4-4e26-bfb4-1bce57fd0c92&mode=build&selected=2275281464-363A6AA2-67CF-4193-9885-A4B79B5D7CDC

File: cmd/wallet-web/src/main.js [hunk 2]
```diff
@@ -5,14 +5,14 @@ SPDX-License-Identifier: Apache-2.0
 */
 
 import { createApp } from 'vue';
-import store from './store';
-import App from './App.vue';
-import router from './router/index';
+import { mapActions, mapGetters } from 'vuex';
```
Comment (talwinder50) [inline cmd/wallet-web/src/main.js:8]: There is technically no change to this file right , i noticed linter keep switching these import, but if we haven't touched the file, shall we still push it ??

**Comment by talwinder50** [comment]
looks good, just minor comment

File: cmd/wallet-web/src/main.js (hunk 2 above)
Comment (birtony) [reply cmd/wallet-web/src/main.js:8]: I've touched it myself, not linter

File: cmd/wallet-web/src/components/Flyout/FlyoutMenu.vue (hunk 1 above)
Comment (birtony) [reply cmd/wallet-web/src/components/Flyout/FlyoutMenu.vue:13]: I just noticed it actually has different widths on these two mock-ups (12rem and 10rem):
image
I'll double check with Mike and cover this in https://github.com/trustbloc/wallet/issues/1302

Comment (talwinder50) [reply cmd/wallet-web/src/components/Flyout/FlyoutMenu.vue:13]: okay

**Pull request merged**